onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -23,7 +23,7 @@
 from .onnx_model import ONNXModel
 
 
-def quantize_data(data, quantize_range, qType):
+def quantize_data(data, quantize_range, qType, symmetric=True):
```

Comment:
>symmetric [](start = 47, length = 9)

Could you please add a setting option for symmetric in quantize

Reply:
Symmetric is added as an extra option.